test/Address.test.js

```diff
@@ -0,0 +1,22 @@
+const AddressImpl = artifacts.require('AddressImpl');
+const SimpleToken = artifacts.require('SimpleToken');
+
+require('chai')
+  .should();
+
+contract('Address', function ([_, creator]) {
```

Comment:
nit: I'd rename `creator` to `anyone`, since we don't really need a creator here.